### DISCUSSION TITLE
Can this project be deployed locally so others can access it remotely afterward? How should I set it up?

### DISCUSSION THREAD

**Discussion by Felixqian4160** [oobabooga/text-generation-webui#1360] [Q&A] [answered]
Can this project be deployed locally so others can access it remotely afterward? How should I set it up?

**Comment by DanielusG** [comment]
If i well understand, you can use flag --share on server startup, so you can access it remotely

**Reply by Felixqian4160** [reply]
# Launch the web UI
create_interface()
while True:
    time.sleep(0.5)
    if shared.need_restart:
        shared.need_restart = False
        shared.gradio['interface'].close()
        create_interface()
False chang to True？？

**Reply by Felixqian4160** [reply]
--share use this？
Could not create share link. Please check your internet connection or our status page: https://status.gradio.app
this is the result

**Reply by DanielusG** [reply, marked as answer]
no, when you launch the webui, you use 
`python server.py --share`

**Reply by Felixqian4160** [reply]
thank you I will try
